test/recipes/60-test_x509_test.t

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env perl
+# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+use OpenSSL::Test::Simple;
+
+simple_test("test_x509_test", "x509_test");
```

Comment:
I'd call the test from the 25-test_x509.t recipe instead. It also needs to be skipped when ec is disabled - this should resolve the CI failure.

Reply:
Done